### DISCUSSION TITLE
Project 1 Feedback

### DISCUSSION THREAD

**Discussion by anbadrin** [airavata-courses/neo#71] [General]
I tried to follow the steps in the readme and run the project. The steps are really compact and the use of docker-compose makes it really easy to run. The project runs fine but the response time is very high. Other than that, the UI and the code looks great.

Team Orion

**Comment by niravraje** [comment]
Thanks Team Orion for your feedback! 
The longer response time is something we are aware of and we'll investigate and optimize the same for our next release. For now, we are recommending reviewers increase their Docker resource limit settings for convenience. (Currently, response time is ~15 seconds with 6 GB RAM permitted for Docker) We'll definitely look into how we can optimize this further.

Thanks,
Nirav
Team Neo
